pkg/apis/core/v1beta1/helper/errors.go

```diff
@@ -223,6 +223,23 @@ type WrappedLastErrors struct {
 	LastErrors  []gardencorev1beta1.LastError
 }
 
+// DeprecatedNewWrappedLastErrors returns an error
+func DeprecatedNewWrappedLastErrors(description string, err error) *WrappedLastErrors {
+	var lastErrors []gardencorev1beta1.LastError
+
+	for _, partError := range utilerrors.Errors(err) {
+		lastErrors = append(lastErrors, *LastErrorWithTaskID(
+			partError.Error(),
+			utilerrors.GetID(partError),
+			DeprecatedDetermineErrorCodes(partError)...))
+	}
+
+	return &WrappedLastErrors{
+		Description: description,
+		LastErrors:  lastErrors,
+	}
+}
+
```

Comment:
This seems to be not used in extensions, so you could double check and drop it.

Reply:
I am also not 100% sure if it is not being used anywhere else, so let's keep it for now and it can be dropped with the dropping of other deprecated functions.